frontend/src/features/admin-form/settings/components/MultiLanguageSection/FormMultiLanguageToggle.tsx

```diff
@@ -0,0 +1,202 @@
+  unicodeLocale,
+  isDefaultLanguage,
+  isLast,
+}: LanguageTranslationRowProps): JSX.Element => {
+  const { data: settings } = useAdminFormSettings()
+
+  const supportedLanguages = settings?.supportedLanguages ?? null
+
+  const [isLanguageSupported, setIsLanguageSupported] = useState(
+    supportedLanguages?.indexOf(unicodeLocale) !== -1,
+  )
+
+  const languageToDisplay = convertUnicodeLocaleToLanguage(unicodeLocale)
+
+  const { mutateFormSupportedLanguages } = useMutateFormSettings()
+
+  const handleToggleSupportedLanague = useCallback(
+    (language: Language) => {
+      if (supportedLanguages == null) return
+
+      // Remove support for this language if it exists in supportedLanguages
+      const existingSupportedLanguageIdx = supportedLanguages.indexOf(language)
+      if (existingSupportedLanguageIdx !== -1) {
+        supportedLanguages.splice(existingSupportedLanguageIdx, 1)
+        setIsLanguageSupported(false)
+        return mutateFormSupportedLanguages.mutate({
+          nextSupportedLanguages: supportedLanguages,
+          selectedLanguage: language,
+        })
+      }
+
+      // If selected language is not in supportedLanguages then add this language
+      // to supportedLanguages
+      else {
+        const updatedSupportedLanguages = supportedLanguages.concat(language)
+        setIsLanguageSupported(true)
+        return mutateFormSupportedLanguages.mutate({
+          nextSupportedLanguages: updatedSupportedLanguages,
+          selectedLanguage: language,
+        })
+      }
+    },
+    [mutateFormSupportedLanguages, supportedLanguages],
+  )
+
+  return (
+    <>
+      <Flex alignItems="center" w="100%" py={2}>
+        <Flex marginRight="auto">
+          <Text mr="0.75rem">{languageToDisplay}</Text>
+          {isDefaultLanguage && (
+            <Badge variant="solid" colorScheme="success" color="secondary.700">
+              Default
+            </Badge>
+          )}
+        </Flex>
+        {!isDefaultLanguage && (
+          <HStack spacing="0.75rem">
+            <IconButton
+              variant="clear"
+              icon={
+                isLanguageSupported ? (
+                  <GoEye width="44px" />
+                ) : (
+                  <GoEyeClosed width="44px" />
+                )
+              }
+              colorScheme="secondary"
+              aria-label={`Select ${unicodeLocale} as the form's default language`}
+              onClick={() => handleToggleSupportedLanague(unicodeLocale)}
+            />
+            <IconButton
+              variant="clear"
+              icon={<BiEditAlt width="44px" />}
+              colorScheme="secondary"
+              aria-label={`Add ${unicodeLocale} translations`}
+              // TODO: Will add redirection to translation section in next PR
+              onClick={() => console.log('Edit translations')}
+            />
+          </HStack>
+        )}
+      </Flex>
+      {!isLast && <Divider />}
```

Comment:
formsg has a convention to use 
```{!!isLast ? <Divider /> : null}``` instead of using `&&`

Reply:
I understand. Just out of curiosity is there any reason why the convention is preferred over using `&&`?

Reply:
I'm not sure, I think it was a practice that has been applied before throughout the codebase (due to the belief that it is more readable) and hence just maintained for overall consistency in the codebase

Reply:
Sorry I, just saw this. 

My reason for avoiding `&&` on the repo:

This prevents us from having to invest into investigating really tricky situations where the value might be erroneously rendered. `!isLast` is definitely a boolean in this particular case, but we can never assure that another developer would use `isLast` in the next PR. To avoid such a possibility, we can refrain from using this as much as possible.

This is generally considered a footgun in my circle (even though react has evolved a lot since, and typing systems are also getting better at falsy values that will be rendered or not).